ovos_phal_plugin_ipgeo/__init__.py

```diff
@@ -57,25 +59,12 @@ def on_reset(self, message=None):
 
     @staticmethod
     def ip_geolocate(ip=None):
-        if not ip or ip in ["0.0.0.0", "127.0.0.1"]:
-            ip = requests.get('https://api.ipify.org').text
-        fields = "status,country,countryCode,region,regionName,city,lat,lon,timezone,query"
-        data = requests.get("http://ip-api.com/json/" + ip,
-                            params={"fields": fields}).json()
-        region_data = {"code": data["region"],
-                       "name": data["regionName"],
-                       "country": {
-                           "code": data["countryCode"],
-                           "name": data["country"]}}
-        city_data = {"code": data["city"],
-                     "name": data["city"],
-                     "state": region_data}
-        timezone_data = {"code": data["timezone"],
-                         "name": data["timezone"]}
-        coordinate_data = {"latitude": float(data["lat"]),
-                           "longitude": float(data["lon"])}
-        return {"city": city_data,
-                "coordinate": coordinate_data,
-                "timezone": timezone_data}
-
-
+        try:
+            # configured backend may throw some errors if its down
+            api = GeolocationApi()
+            return api.get_ip_geolocation(ip)
+        except Exception as e:
```

Comment:
For future dev efforts, we should log the exception, so we can be more aware of what issues may come up and handle them properly